readme.md

```diff
@@ -45,12 +45,10 @@ mod bindings {
 }
 
 use bindings::{
-      Windows::Win32::WindowsProgramming::CloseHandle,
-      Windows::Win32::WindowsAndMessaging::MessageBoxA,
-      Windows::Data::Xml::Dom::*,
-      Windows::Win32::SystemServices::{
-          CreateEventW, SetEvent, WaitForSingleObject
-      },
+    windows::data::xml::dom::*,
+    windows::win32::system_services::{CreateEventW, SetEvent, WaitForSingleObject, PWSTR},
+    windows::win32::windows_and_messaging::{MessageBoxA, HWND, MB_FLAGS},
+    windows::win32::windows_programming::CloseHandle,
```

Comment:
Same here (see [this pull request](https://github.com/microsoft/windows-rs/pull/646)).